libs/ledger-live-common/src/exchange/swap/hooks/useProviderRates.test.ts

```diff
@@ -330,6 +333,34 @@ describe("useProviderRates", () => {
     expect(result.current.rates.value).toBeUndefined();
   });
 
+  it("SwapExchangeRateAmountTooLowOrTooHigh should take precedence over SwapExchangeRateAmountTooLow", async () => {
+    const swapExchangeRateAmountTooLowError = new SwapExchangeRateAmountTooLow(
+      undefined,
+      {
+        amount: new BigNumber(1),
+      }
+    );
+
+    const swapExchangeRateAmountTooLowOrTwoHighError =
```

Comment:
Small thing: should be `...TooHighError`

Reply:
It can be too small too. (I have tested it with 0.000...01 and works in the same way.